datalake-api/src/main/java/com/sequenceiq/sdx/api/endpoint/SdxEndpoint.java

```diff
@@ -317,4 +317,10 @@ SdxRecommendationResponse getRecommendation(
             @NotNull(message = "The 'cloudPlatform' query parameter must be specified.") @QueryParam("cloudPlatform") String cloudPlatform,
             @NotNull(message = "The 'region' query parameter must be specified.") @QueryParam("region") String region,
             @QueryParam("availabilityZone") String availabilityZone);
+
+    @GET
+    @Path("crn/{crn}/internal/stoppable")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = "Determines if the datalake can be stopped", nickname = "isStoppable")
+    boolean isStoppableInternal(@PathParam("crn") String crn, @QueryParam("initiatorUserCrn") String initiatorUserCrn);
```

Comment:
Wouldn't it make sense to return a more sophisticated object (`SdxStopValidationResponse` for example) which describes why the data lake is not stoppable.